docs/docs/building-modules/02-messages-and-queries.md

```diff
@@ -38,6 +38,40 @@ Each `Msg` service method must have exactly one argument, which must implement t
 
 `sdk.Msg` interface is a simplified version of the Amino `LegacyMsg` interface described [below](#legacy-amino-msgs) with the `GetSigners()` method. For backwards compatibility with [Amino `LegacyMsg`s](#legacy-amino-msgs), existing `LegacyMsg` types should be used as the request parameter for `service` RPC definitions. Newer `sdk.Msg` types, which only support `service` definitions, should use canonical `Msg...` name.
 
+`sdk.Msg` is a alias of `proto.Message`. To attach a `ValidateBasic()` method to a message then you must add methods to the type adhereing to the `HasValidateBasic`.
```

Comment:
This (and L30) seem really out of place imho, imho we should add a subcategory for it.
Additionally, the legacy-amino-msgs category contains outdated information.